src/components/Login.js

```diff
@@ -45,7 +45,7 @@ function LoginForm(props) {
       .then(response => {
         setLoadingUser(false);
         localStorage.setItem("token", response.data.token);
-        props.history.push("/");
```

Comment:
removed by accident?

Reply:
I updated to props.history.push("/facilityList"); Is that ok? ( bearing in mind comment below about future routing)